src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -195,6 +195,28 @@ class_exists('Test\\'.__NAMESPACE__.'\\'.$class, true);
         $this->assertSame($xError, $lastError);
     }
 
+    public function testInterfaceExtendsDeprecatedInterface()
+    {
+        set_error_handler('var_dump', 0);
+        $e = error_reporting(0);
+        trigger_error('', E_USER_DEPRECATED);
+
+        class_exists('Test\\'.__NAMESPACE__.'\\NonDeprecatedInterfaceClass', true);
+
+        error_reporting($e);
+        restore_error_handler();
+
+        $lastError = error_get_last();
+        unset($lastError['file'], $lastError['line']);
+
+        $xError = array(
+            'type' => E_USER_DEPRECATED,
+            'message' => '',
+        );
+
+        $this->assertSame($xError, $lastError);
+    }
+
     public function provideDeprecatedSuper()
```

Comment:
please keep this dataProvider near the test using it instead of inserting your new test between them